images.go

```diff
@@ -11,9 +11,10 @@ import (
 
 // Image defines the data we process about a docker image.
 type Image struct {
-	Name    string `yaml:"name"`
-	Comment string `yaml:"comment,omitempty"`
-	Tags    []Tag  `yaml:"tags"`
+	Name         string `yaml:"name"`
+	Comment      string `yaml:"comment,omitempty"`
+	OverrideName string `yaml:"overrideName,omitempty"`
```

Comment:
Would `OverrideRepoName` make more sense? I feel like `Name` and `OverrideName` is a bit confusing

Reply:
LGTM